service/listener.py

```diff
@@ -68,8 +68,10 @@ def start_listening(self):
                 description = f"{package_name} created"
                 status = "finished"
             except Exception as e:
-                status = "failed"
-                description = f"Creation of package failed\n{str(e)}"
+                status = "finished"
+                description = f"Creation of package for {bundle_name} failed\n{str(e)}"  # noqa
+                import traceback
```

Comment:
Requires `from nxtools import log_traceback` at the top, but then you can do `log_traceback(e)`